### PR TITLE
Added query of images for published versions

### PR DESCRIPTION
**Pull request by hananscribe**

### CODE REVIEW

File: Scribe_API.postman_collection.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
 	"info": {
 		"_postman_id": "a347be16-903a-48c2-b96f-565267fb13b3",
 		"name": "Scribe API",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "30094260"
```
Comment (cppvik) [inline Scribe_API.postman_collection.json]: What's `_exporter_id` here and do we actually need it?

File: Scribe_API.postman_collection.json [hunk 2]
```diff
@@ -514,6 +515,33 @@
...
 			"response": []
+		},
+		{
+			"name": "Query Images in published versions",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n  \"superset_token\": \"{{access_token}}\",\n  \"query\": {\n       \"datasource\": {\n            \"id\": {{dataset_API_Images}},\n            \"type\": \"table\"\n        },\n       \"queries\": [\n            {\n               \"columns\": [\n                    \"image_id\",\n                    \"targetName\",\n                    \"logical_app\",\n                    \"logical_app_version\"\n                ],\n                \"filters\": [\n                    {\n                        \"col\": \"version_published\",\n                        \"op\": \"==\",\n                        \"val\": true \n                    }\n                ],\n                 \"metrics\": [],\n                \"row_limit\": 0\n            }\n        ],\n        \"result_format\": \"json\",\n        \"result_type\": \"results\"\n    }\n}",
```
Comment (cppvik) [inline Scribe_API.postman_collection.json:525]: I believe columns should be

```
 "columns": [
    "image_id",
    "targetName",
    "logical_app",
    "logical_app_version",
    "version_published"
],
```

Comment (hananscribe) [reply Scribe_API.postman_collection.json:525]: they are...

Comment (cppvik) [reply Scribe_API.postman_collection.json:525]: Sorry, my bad

**Review by cppvik — DISMISSED** [review]

File: Scribe_API.postman_collection.json (hunk 1 above)
Comment (hananscribe) [reply Scribe_API.postman_collection.json]: removed

**Pull request merged**